x/protorev/genesis.go

```diff
@@ -19,29 +19,32 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	k.SetProtoRevEnabled(ctx, genState.Params.Enabled)
 	k.SetDaysSinceModuleGenesis(ctx, 0)
 	k.SetLatestBlockHeight(ctx, uint64(ctx.BlockHeight()))
-	k.SetRouteCountForBlock(ctx, 0)
+	k.SetPointCountForBlock(ctx, 0)
 
-	// Configure max routes per block. This roughly correlates to the ms of execution time protorev will
+	// Configure max pool points per block. This roughly correlates to the ms of execution time protorev will
 	// take per block
-	if err := k.SetMaxRoutesPerBlock(ctx, 100); err != nil {
+	if err := k.SetMaxPointsPerBlock(ctx, 100); err != nil {
```

Comment:
Note to reviewers: We're still deciding pool weights by adding more tests and doing an analysis of on-chain data for # of swaps per tx/block in the past so we can get estimates of what to be expected.